docs/install.md

```diff
@@ -12,10 +12,16 @@
     - [Install Poetry in Windows](#install-poetry-in-windows)
   - [FAQ and Troubleshooting](#faq-and-troubleshooting)
 
+
+LISA can be installed on any computer without having to be installed on each tested node. It can be installed on tested nodes to perform local testing. It can connect to another tested node using SSH. The following are examples showing where LISA is installed.
+
```

Comment:
Rewrap.

Reply:
Fixed. But I'm wondering how it can help reading. It doesn't effect the rendered markdown result, and the source code can be wrapped by modern editors.

Reply:
It is wrapped, but a pain to follow. Not everyone codes in fullscreen, that's why people have max line standards (we have ours at 88, in Python). They'd rather have multiple buffers side-by-side at all times, only focusing on something full screen when it makes sense (and then even at those times using less than the actual full screen in width).